Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6311,6 +6311,20 @@ ScreenOrientationLockingAPIEnabled:
     WebCore:
       default: false
 
+ScreenTimeEnabled:
+  type: bool
+  status: unstable
+  condition: ENABLE(SCREEN_TIME)
+  humanReadableName: "Screen Time"
+  humanReadableDescription: "Enable Web ScreenTime"
```

Comment:
I'm not sure what the word Web is buying us here, given this is WebKit. Also, let's pick one spelling/spacing of Screen Time (ideally the one used in e.g. Settings)